packages/runtime/types/features.ts

```diff
@@ -0,0 +1,15 @@
+import { z } from 'zod'
+
+export type MakeswiftFeaturesConfig = {
+  versioning?: boolean
+}
+
+export type MakeswiftFeatures = {
+  [K in keyof MakeswiftFeaturesConfig]-?: Exclude<MakeswiftFeaturesConfig[K], undefined>
+}
```

Comment:
When using Zod, I think we should always infer the types from the schema to avoid a mistake where our type doesn't match Zod's TypeScript types